### PR TITLE
Use LibCURL_ jll and fix SSL cert problem

### PR DESCRIPTION
**Pull request by aviks**
Incorporates #75, and fixes the SSL certificate problem by supplying the mozilla CA bundle with the jll.  

[Edit] I've removed the `@test_broken` annotation for the ssl tests in windows and bsd. Tests should now pass on all platforms.

### CODE REVIEW

**Comment by aviks** [comment]
FreeBSD test failure seems unrelated. Works for windows and linux.

File: Project.toml [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 
 [deps]
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
-Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+LibCURL_jll = "deac9b47-8bc7-5906-a0fe-35ac56dc84c0"
 
 [compat]
-BinaryProvider = "0.5"
-julia = "0.7, 1.0"
+LibCURL_jll = "7.68"
```
Comment (geokollias) [inline Project.toml:9]: Could we bump that up to 7.70 to pick HTTP/2 (`nghttp2`) support as well?

Comment (aviks) [reply Project.toml:9]: Ah yes, I was about to do that.

**Comment by aviks** [comment]
@omus let me know if you have any concerns with this. I'll merge and tag this soon-ish otherwise.

**Comment by omus** [comment]
This looks great! I'll just check the failing FreeBSD test and go from there. I think what I'll do is I'll merge my open PR and then we can rebase these changes. That should make it more obvious what is going on when looking at the merged/closed PRs.

**Comment by aviks** [comment]
Incorporated into #75

**Pull request closed without merging**